.github/PULL_REQUEST_TEMPLATE/release_pr_template.md

```diff
@@ -0,0 +1,12 @@
+# Release Checklist
```

Comment:
Does github use the files here for actual PR templates when we open it manually in the UI?

Reply:
_Sometimes._ The stuff in there can be automatically put in via the `template` query param, but that query param is only respected when using the "web" API. The REST API ignores it, and if you use the web api, it brings you to the "finishing editing this PR and click submit screen", so not useful for our purposes. That's why `.github/pull_request_template.md` still exists, because that's the default one to use when others aren't specified.